Modules/Sources/WordPressShared/WKWebView+UserAgent.swift

```diff
@@ -0,0 +1,25 @@
+import CocoaLumberjackSwift
+import WebKit
+
+@objc
+public extension WKWebView {
+    static let userAgentKey = "_userAgent"
+
+    /// Call this method to get the user agent for the WKWebView
+    @objc
+    func userAgent() -> String {
+        guard let userAgent = value(forKey: WKWebView.userAgentKey) as? String, !userAgent.isEmpty else {
+            // TODO: Original implementation logged a message to Tracks/Sentry
+            DDLogError("This method for retrieveing the user agent seems to be no longer working.  We need to figure out an alternative.")
```

Comment:
https://github.com/wordpress-mobile/WordPress-iOS/blob/a6eaa7aa8acb50828449df2d3fccaa50d7def821/WordPress/Classes/Extensions/WKWebView%2BUserAgent.swift

Reply:
It sounds like a job for `wpAssertionFailure`.

Reply:
Possibly. 

But `wpAssertionFailure` depends on Sentry and Tracks extracting that is it's own can of worms... 😞